### DISCUSSION TITLE
Default carousel slider / brand slider / testimonial slider in breeze

### DISCUSSION THREAD

**Discussion by sdomeradzki** [breezefront/community#14] [3. Integrations]
Hi,
If possible in Breeze set a any slider like in attachment?
In most popular themes this solution is as default, but I can't find solution for breeze.

I see that in extensions from SwissLab is possible set only big slider, but not carousel slider with image next to image. (please look on attachmet).

<img width="497" alt="Zrzut ekranu 2022-09-22 o 15 38 48" src="https://user-images.githubusercontent.com/110283346/191763032-e59524fb-a29e-4afc-b75c-f04bc6c3b911.png">

**Comment by vovayatsyuk** [comment]
Yes. Use raw html code as shown in this guide: https://breezefront.com/cms-elements#carousel-slider

**Comment by sdomeradzki** [comment]
Oh, I see now! I don't know where I missing my eyes .. Sorry for that!
Thanks!
